backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/theme/BpkTheme.kt

```diff
@@ -37,34 +48,76 @@ fun BpkTheme(
   content: @Composable () -> Unit,
 ) {
   val typography = BpkTypography(defaultFontFamily = fontFamily)
+  val colors = if (isSystemInDarkTheme()) BpkColors.dark() else BpkColors.light()
+
   CompositionLocalProvider(
     LocalBpkTypography provides typography,
+    LocalBpkColors provides colors,
+    LocalElevationOverlay provides null,
```

Comment:
Elevation overlay modifies the surface colour by adding overlay colour to it. In Backpack, we control the elevation shases manually, thus not overlay is needed